src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```diff
@@ -33,7 +33,7 @@
     "ajv-formats": "^2.1.1",
     "axios": "^0.24.0",
     "dot-object": "^2.1.4",
-    "immutability-helper": "2.7.0",
+    "immutability-helper": "2.9.1",
```

Comment:
Breaks build. We should bump this (or really get rid of the need for this package altogether). I see that altinnpedia references the issue https://github.com/Altinn/altinn-studio/issues/3952 , but this is closed. Suggest handling bumping this as a seperate issue. This dep is part of our execption overview on pedia.

Reply:
Will revert.

Reply:
I missed it in the exceptions doc - thx for checking twice:)